docker/supervisord-services.ini

```diff
@@ -0,0 +1,16 @@
+[program:nginx]
+command=/usr/sbin/nginx -c /etc/nginx/nginx.conf
+stdout_logfile=/dev/stdout # will only be effective if the program is also logging to stdout
+stderr_logfile=/dev/stderr
+
+[program:uwsgi]
+command=/usr/sbin/uwsgi --ini /home/app/django/docker/cla_backend.ini --die-on-term --log-master
```

Comment:
🙋‍♂ `--die-on-term` is to enable SIGTERM to kill the uwsgi processes, which is preferable with Docker as stopping containers behave as [follows](https://docs.docker.com/engine/reference/commandline/stop/#extended-description):

> The main process inside the container will receive SIGTERM, and after a grace period, SIGKILL.